### PR TITLE
fix: enabling linting and fixing errors

### PR DESCRIPTION
**Pull request by tripodsan**
testing tests on fork

URL for testing:

- https://testing-tests--helix-website--tripodsan.hlx.page/

### CODE REVIEW

File: tools/sidekick/app.js [hunk 1]
```diff
@@ -11,8 +11,6 @@
  */
 /* eslint-disable no-console, no-alert */
 
-'use strict';
```
Comment (rofe) [inline tools/sidekick/app.js:14]: This is just a distribution copy. I'll change this in `helix-sidekick`.

File: tools/sidekick/module.js [hunk 2]
```diff
@@ -11,8 +11,6 @@
  */
 /* eslint-disable no-console, no-alert */
 
-'use strict';
```
Comment (rofe) [inline tools/sidekick/module.js:14]: This is just a distribution copy. I'll change this in `helix-sidekick`.

File: tools/sidekick/module-hlx2.js [hunk 3]
```diff
@@ -11,8 +11,6 @@
  */
 /* eslint-disable no-console, no-alert */
 
-'use strict';
```
Comment (rofe) [inline tools/sidekick/module-hlx2.js:14]: This file is no longer needed. I'll remove it in a separate PR.

File: .github/workflows/lint.yml [hunk 4]
```diff
@@ -0,0 +1,28 @@
+name: Lint scripts and CSS
```
Comment (rofe) [inline .github/workflows/lint.yml:1]: Can we wrap this back into `test.yaml` to align with the boilerplate?

**Review by rofe — CHANGES_REQUESTED** [review]
Thanks @tripodsan for debugging this and cleaning up the code! Just one minor thing...

File: .github/workflows/lint.yml (hunk 4 above)
Comment (tripodsan) [reply .github/workflows/lint.yml:1]: I kind of like the 2 actions - as they address 2 separate issues... but ok. if you like.

**Review by rofe — APPROVED** [review]

**Pull request merged**